src/tests/multihost/ad/conftest.py

```diff
@@ -302,18 +314,17 @@ def adjoin(session_multihost, request):
 
     def _join(membersw=None):
         """ Join AD """
+        # We are using dc instead or realm to mitigate impact of AD being
```

Comment:
Can you please add a comment stating the expected and non-expected behavior, "realm join REALM.COM should be automatically discovered if it was on the same network, blah blah blah but due to idmci resources residing on different networks, blah blah blah" 

Let's also mention it to the team next week, this is a moderate step away from functional testing and we should all know about it.